README.md

```diff
@@ -10,52 +10,144 @@ We need very broad coverage of news outlets. Contributed spiders are very welcom
 
 It's really easy to contribute spiders. Basically you can copy an existing spider and change the xpaths to find the elements we're extracting.
 
-See the In Progress column at [https://trello.com/b/9TVRB4gb/public-people](https://trello.com/b/9TVRB4gb/public-people) to see which publications are currently being tackled to avoid duplication.
+See the _In Progress_ column at [https://trello.com/b/9TVRB4gb/public-people](https://trello.com/b/9TVRB4gb/public-people) to see which publications are currently being tackled to avoid duplication.
 
 Next, go get started at [Development](#development)
 
 ## Copyright of news content
 
 We do not make news content available for public consumption. We simply store and index the news content and the original URL and publication date to provide search functionality similar to search engines. This project intends to provide better access to news on the publisher's website. It should be used to send readers to relevant news websites rather than to replace them.
 
+## Requirements
+
+- Python 3.5+
+- Python virtual environment with packages installed from [requirements.txt](/requirements.txt). This main library used [Scrapy](https://scrapy.org/).
+- Setup [scrapyd](https://scrapyd.readthedocs.io/en/stable/) (optional).
+- Aleph account and credentials for uploading results (optional).
+
 ## Development
 
 ### Set up your development environment
 
+#### Clone
+
 Fork this repository on GitHub and clone your fork:
+
 ```bash
-git clone https://github.com/your-name/scrape-news.git
+$ git clone https://github.com/your-name/scrape-news.git
```

Comment:
The problem with these $s is that you can't just copy and paste instructions. It's already a code block: it's clear it's code.

```suggestion
git clone https://github.com/your-name/scrape-news.git
```